motionplan/collision.go

```diff
@@ -181,3 +194,21 @@ func (cg *collisionGraph) collisions() []Collision {
 func (cg *collisionGraph) addCollisionSpecification(specification *Collision) {
 	cg.setDistance(specification.name1, specification.name2, math.NaN())
 }
+
+func createUniqueCollisionMap(geoms []spatial.Geometry) (map[string]spatial.Geometry, error) {
+	unnamedCnt := 0
+	geomMap := map[string]spatial.Geometry{}
+
+	for _, geom := range geoms {
+		label := geom.Label()
+		if label == "" {
+			label = unnamedCollisionGeometryPrefix + strconv.Itoa(unnamedCnt)
```

Comment:
I don't think there's strictly a problem here, but if `x` and `y` are the same above, could we get different labels in each map to the same geometries?

Reply:
Specifically if we had multiple `""` labeled geometries in the original list.

Reply:
No, because we're iterating over a list, not over a map. They would always be at the same index.